src/com/ra4king/circuitsim/gui/CircuitSim.java

```diff
@@ -472,12 +472,20 @@ String getCircuitName(CircuitManager manager) {
 		
 		return null;
 	}
+
+    public Map<String, CircuitManager> getCircuitManagers() {
+		return circuitManagers.keySet().stream()
```

Comment:
Use .entrySet() instead, it's more efficient.

Reply:
Also do you care about the order of the tabs? You don't need the last 2 parameters if you don't.

Reply:
was copypasting from above, just want the grader to work haha